### ISSUE TITLE
Placeholders still not visible in v2.5.0

### ISSUE THREAD

**Issue by HannesOlszewski** [rjsf-team/react-jsonschema-form#2271]
Hello there,

I installed the latest version (2.5.0) of `@rjsf/core` and `@rjsf/bootstrap-4`.

As I can see in the releases section version 2.5.0 fixed the placeholder issue. But in the package installed through yarn the issue is still present (also when using npm instead of yarn).

I checked the files in `node_modules` and the placeholder prop is not present for the TextWidget, although the `package.json` says it's version 2.5.0.

Is there an issue with the published package or is it an issue on my end?

**Comment by epicfaace** [comment]
Hmm, is it working on the playground (https://rjsf-team.github.io/react-jsonschema-form/)?

**Comment by HannesOlszewski** [comment]
Funny thing, in the playground with the `bootstrap-4` theme, a normal text field has a placeholder, but when adding `"ui:widget": "password"` it has no placeholder (same when selecting `semantic-ui`, the other themes work completely in the playground though).

But when using the (working) schemas from the playground in my local installation there are no placeholders.

My form schema:
```json
{
  "title": "Login",
  "type": "object",
  "properties": {
    "username": {
      "type": "string",
      "title": "Username",
      "propertyOrder": 1
    },
    "password": {
      "type": "string",
      "title": "Password",
      "widget": "password",
      "propertyOrder": 2
    }
  },
  "required": [
    "username",
    "password"
  ]
}
```

My ui schema:
```json
{
  "username": {
    "ui:placeholder": "Username"
  },
  "password": {
    "ui:placeholder": "Password",
    "ui:widget": "password"
  }
}
```

When using `material-ui` locally, the placeholders work though, but not with `bootstrap-4`.

**Comment by epicfaace** [comment]
Looks like when I built 2.5.0, the build failed but I didn't notice, so I ended up publishing an older version to npm. Working on a fix...

**Comment by epicfaace** [comment]
@HannesOlszewski this should now be fixed in [v2.5.1](https://github.com/rjsf-team/react-jsonschema-form/releases/tag/v2.5.1). We've added automated publishing via GitHub Actions so this shouldn't happen again. Thanks for your patience!
